lib/open-sesame.coffee

```diff
@@ -68,3 +72,5 @@ module.exports =
 
   activate: (state) ->
     atom.commands.add 'atom-text-editor', OPEN_COMMAND, openFileUnderCursor
+    atom.views.getView(atom.workspace).addEventListener 'click', (event) ->
+      openFileUnderCursor() if event.altKey or event.metaKey
```

Comment:
I see what you mean about this behaving oddly. Since it doesn't work too well, can we remove it and scope it out of this PR? I'm sure this functionality is doable, but I think there's probably a nicer way to do it where maybe the text becomes underlined and you get a `cursor: pointer` so that it basically looks and behaves as a link.